201-vm-dedicated-hosts/azuredeploy.parameters.json

```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "dhgNamePrefix": {
```

Comment:
can we remove all of these and just use the defaultValues in the template?  or do you want to test different defaults?

Reply:
Since all params have defaults, we can remove them. However, if someone does not know how to add a param to the azuredeploy.parameters.json file, they won't be able to pick it up easily

Reply:
There are plenty of samples that show this and most folks beginning are using UI.  Common practice for the repo is to use defaultValues where possible (which is a better ux than adding the param) and the just use the param file to direct CI, primarily for the GEN values.